entgql/annotation.go

```diff
@@ -125,6 +126,19 @@ func (a *Annotation) Decode(annotation interface{}) error {
 	return json.Unmarshal(buf, a)
 }
 
+// DecodeAnnotation decodes the annotation from the schema.
+func DecodeAnnotation(annotations gen.Annotations) (*Annotation, error) {
+	ant := &Annotation{}
+	if annotations == nil || annotations[ant.Name()] == nil {
+		return ant, nil
+	}
+
```

Comment:
not a big deal, but I usually try to reduce the amount of newlines/spaces if the blocks handle the same case. 
```suggestion
```

Reply:
I don't want to save lines of code, because I want to ensure the readability of the source code.